notification/README.md

```diff
@@ -16,9 +16,9 @@ Use `npm test` to run the unit tests.
 
 ## Running end-to-end tests
 
-Run integration tests in docker via the following command:
+Run integration tests in docker via the following command in the project root directory (open-data-service):
```

Comment:
any special reason why you recommend to start the notification-it from the root directory, but all other services from their own?

A quick test showed me that starting the notification-it from the subdirectory works as well (if you do it in similar style to the others)

Reply:
> any special reason why you recommend to start the notification-it from the root directory, but all other services from their own?
> 
> A quick test showed me that starting the notification-it from the subdirectory works as well (if you do it in similar style to the others)

Good point. I will adjust the notification README to be consistent with the other services.